scholarx/archive/functions.js

```diff
@@ -2,6 +2,10 @@ $(function () {
     loadNavAndFooter('/assets/content/static');  //relative path to content directory
 });
 
+//global variables to hold mentor and mentee data
+var mentor2019, mentor2020, mentor2021;
+var mentee2019, mentee2020, mentee2021;
```

Comment:
we can use `let` here.

Reply:
@Madhawa97 Done, Any other changers madhawa?

Reply:
So `var` is a global variable right? You put the comment as `global variables to hold mentor and mentee data` bcs of the `var` variable no? So should it change to `variables to hold mentor and mentee data` @janithlahirukariyawasam